src/k8s-engine/lima.ts

```diff
@@ -683,7 +683,8 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
       // Trigger kuberlr to ensure there's a compatible version of kubectl in place for the users
       // rancher-desktop mostly uses the K8s API instead of kubectl, so we need to invoke kubectl
       // to nudge kuberlr
-      await childProcess.spawnFile(resources.executable('kubectl'), ['cluster-info'],
+      await childProcess.spawnFile(resources.executable('kubectl'),
+        ['--context', 'rancher-desktop', 'cluster-info'],
```

Comment:
Hmm, we should have the context name be a constant somewhere… mostly to make it easier to identify where we use the context.

Currently, it's hard-coded:
https://github.com/rancher-sandbox/rancher-desktop/blob/50d11c8dc3225307ec3a5e335ec09c02a7517f77/src/k8s-engine/k3sHelper.ts#L532

Reply:
+1, but that should wait for a tech-debt issue to add a module of global constants.